iterx.go

```diff
@@ -180,7 +170,13 @@ func (iter *Iterx) scanAll(dest interface{}, structOnly bool) bool {
 
 		// allocate memory for the page data
 		if !alloc {
-			v = reflect.MakeSlice(slice, 0, iter.Iter.NumRows())
+			sz := iter.NumRows()
+			if sz == 0 {
+				// Taken out of the air.
+				// Should it be the default page size?
+				sz = 100
```

Comment:
Note that users may change that https://godoc.org/github.com/gocql/gocql#Query.PageSize so it should be a default unless changed.

Reply:
Not sure exactly why NumRows is wrong, maybe it's an gocql error?